bazel/utils/container/muk/muk.py

```diff
@@ -154,7 +154,7 @@ def _run_cmd_from_action(action: mpb.Action) -> str:
         yum_install = action.yum_install
         run_cmd = """\
 RUN TZ=UTC yum update -y && \\
-    TZ=UTC yum install -y \\
+    TZ=UTC yum install --allowerasing --enablerepo=devel -y \\
```

Comment:
Do you want to do this for both distroes, or maybe isolate it to rocky?

Reply:
I'll isolate it to Rocky. Good suggestion.